mandatory/2-function-creation.js

```diff
@@ -4,7 +4,9 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+function isAcceptableUser(userAge, isLoggedIn) {
```

Comment:
Another great answer.

How might you refactor this? Try writing this function a few different ways and thinking about it (this is also fun to do in Codewars).